.npmrc

```diff
@@ -1 +1,40 @@
+node-version=18.14.1
+
+# Not always possible to be strict, but if it works for you, keep it to true.
+# https://pnpm.io/next/npmrc#strict-peer-dependencies
 strict-peer-dependencies=false
+
+# Auto install peers should be false to avoid downloading
+# extraneous deps. If the install fails, try first to explicitly add
+# the missing deps in your package. Set it to true at last resort
+# (when the problem comes from upstream dependencies). The best is false.
+# https://pnpm.io/npmrc#auto-install-peers
+auto-install-peers=false
+
+# Helps locating duplicates, default in v8
+# https://pnpm.io/next/npmrc#use-lockfile-v6
+use-lockfile-v6=true
+
+# Will fix duplicates due to peer-dependencies (>=7.29.0), default in v8
+# https://github.com/pnpm/pnpm/releases/tag/v7.29.0
+dedupe-peer-dependents=true
+
+# Helps with peer-deps (>=7.23.0), default in v8
+# https://pnpm.io/npmrc#resolve-peers-from-workspace-root
+resolve-peers-from-workspace-root=true
+
+# Default in 8.1.0 to fix issues with root/workspaces hoisting
+# https://pnpm.io/npmrc#dedupe-direct-deps
+dedupe-direct-deps=false
+
+# Pinlock to exact version (default is '^')
+# https://pnpm.io/npmrc#save-prefix
+# see also how save-workspace-protocol affect this https://pnpm.io/npmrc#save-workspace-protocol
+save-prefix=''
+
+# Most of the time, you want to use the rolling protocol for monorepos
+# https://pnpm.io/npmrc#save-workspace-protocol
+save-workspace-protocol=rolling
+
```

Comment:
Why we need this on this PR? If is a general improvement split into another PR

Reply:
🙄